interfaces/HO-Portal/src/app/user/user.page.html

```diff
@@ -39,21 +43,50 @@ <h2>{{ 'page.user.change-password.title'|translate }}</h2>
             >
               <div class="input-item ion-margin-bottom ion-padding-bottom">
                 <label color="medium" class="input-item--label auth-label">{{ 'page.user.change-password.new-password'|translate }}</label>
-                <ion-input
-                  type="password"
-                  autocomplete="new-password"
-                  autocapitalize="off"
-                  clearInput="true"
-                  clearOnEdit="false"
-                  required
-                  [minlength]="8"
-                  name="new-password"
-                  [(ngModel)]="newPassword"
-                  ngDefaultControl
-                  class="ion-margin-vertical">
-                </ion-input>
+                <div class="input-item--content">
+                  <ion-input
+                    type="password"
+                    autocomlete="off"
+                    autocapitalize="off"
+                    clearInput="true"
+                    clearOnEdit="false"
+                    required
+                    [minlength]="8"
+                    name="new-password"
+                    [(ngModel)]="newPassword"
+                    ngDefaultControl
+                    pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}"
+                    class="ion-margin-vertical auth-field">
+                  </ion-input>
+                  <div class="validation-message">
+                      <ion-note color="danger" *ngIf="showShowPasswordMessage()">{{ 'page.user.error-messages.password-length'|translate }}</ion-note>
+                      <ion-note color="danger" *ngIf="showInvalidPasswordMessage()">{{ 'page.user.error-messages.password-characters'|translate }}</ion-note>
+                  </div>
+                </div>
+              </div>
+              <div class="input-item ion-margin-bottom ion-padding-bottom">
+                <label color="medium" class="input-item--label auth-label">{{ 'page.user.change-password.confirm-password'|translate }}</label>
+                <div class="input-item--content">
+                  <ion-input
+                    type="password"
+                    autocomlete="off"
```

Comment:
Here it makes sense, because we don't want that users' browsers/extensions will suggest/fill-in the old password here or something..